### DISCUSSION TITLE
Add alt+tab functionality in full screen

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#223] [Feature Requests]
I would like the Keyboard.lock() API to be implemented when the desktop is in fullscreen. This would enable more keyboard shortcuts such as alt+tab, which would enable the possibility for an alt+tab menu.

**Comment by DustinBrett** [comment]
It actually is enabled when in fullscreen. But I have noticed sometimes it doesn't trigger. I monitor for F11 and when pressed I force them into fullscreen, as the normal F11 alone does not do keyboard lock. This is not so simple but it indeed already exists. As for alt+tab, that would be a feature to add, and its possible.

**Reply by DustinBrett** [reply]
As a test, the Windows key works in fullscreen, which isn't possible without keyboard lock. Also Win+R I have setup.

**Reply by CorruptionVR** [reply]
but what about for devices that do not have F11?

**Reply by DustinBrett** [reply]
You can right click the taskbar to enter fullscreen.

https://github.com/DustinBrett/daedalOS/blob/main/hooks/useGlobalKeyboardShortcuts.ts#L103

**Reply by DustinBrett** [reply]
![image](https://user-images.githubusercontent.com/16656318/198333284-983679de-f1e3-4a16-ad95-7c70645034b6.png)

**Reply by CorruptionVR** [reply]
yes, well, the search key does not work on a Chromebook when I use that method, but the desktop does recognize the search key and opens the start menu when I press another key with it.

**Reply by DustinBrett** [reply]
Well I am linking to the Meta key. If Chromebook doesn't use that mapping, then search really isn't the same as start menu at this point for daedalOS.

**Reply by CorruptionVR** [reply]
this is me pressing the search key, and the A key together
[Video](https://user-images.githubusercontent.com/78228404/198337424-1db2417e-2d39-4b8f-9dd8-5b800eaffbb7.webm)
as you can see, the start menu is opening, but when I press the search key alone, it opens the launcher.

**Reply by CorruptionVR** [reply]
maybe its a security feature of ChromeOS?

**Reply by DustinBrett** [reply]
Ya I am not sure. For me windows key alone can open the menu when keyboard lock is working, based on using the Meta key.

https://www.toptal.com/developers/keycode/for/91
